index.html

```diff
@@ -0,0 +1,55 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="container">
+        <div class="wrapper">
+            <nav class="flex-nav box1">
+                <ul>
+                    <li>
+                        <h1 class="logo">My logo</h1>
+                    </li>
+                    <li class="logo-img">
+                        <i><img src="images/Union.svg" alt="logo"></i>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div class="box2">
+            <div class="title">
+                <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
+            </div>
+
+            <div>
+                <p>I'm a software developer! I can help you build a product, feature or website 
+                    look through some of my work and experience! If you like what you see and have
+                    a project you need coded, don't hestiate to contact me.
+                 </p>
+            </div>
+
+            <div>
+                <p class="connect">
+                    LET'S CONNECT
+                </p>
+                <img src="images/twitter.svg" alt="twitter" class="icon">
+                <img src="images/Linkedin.svg" alt="Linkedin" class="icon">
+                <img src="images/Vector-3.svg" alt="Vector-3" class="icon">
+                <img src="images/Vector-2.svg" alt="Vector-2" class="icon">
+                <img src="images/Vector-4.svg" alt="Vector-4" class="icon">
+            </div>
+
+        </div>
+    </header>
+
+    <section></section>
+
+    <footer></footer>
+    
```

Comment:
- [ ] Kindly😃, try to remove these tags we will use them later in the other sections.💯